citesphere/src/main/webapp/WEB-INF/views/auth/group/items.html

```diff
@@ -599,6 +599,11 @@ <h4 class="panel-title">
 				</a>
 				</span>
 			</td>
+			<td th:data-index="${loop.index}">
```

Comment:
header for the column is missing; also, let's use an arrow to the right or something as icon.

Reply:
do you need the index here for anything?